api/envoy/config/filter/http/aws_lambda/v2/aws_lambda.proto

```diff
@@ -33,6 +35,10 @@ message AWSLambdaPerRoute {
   // When set on a route the filter will not stream data on the encoding step. 
   // Defaults to false.
   bool unwrap_as_alb = 5;
+
+  // transformer configuration used to process response data
+  // cannot be configured simultaneously with unwrap_as_alb
```

Comment:
can we include in the comment that this must be a transformer, maybe include a link to the transformer proto message?

Reply:
Updated that comment here: https://github.com/solo-io/envoy-gloo/pull/167/commits/403b35d31634543f1bcb10d9ff18c58f1818107a